### PR TITLE
enhance: add more background gray colors

### PR DESCRIPTION
**Pull request by marcelgerber**
Soo, when I just went to add more gray background colors, I noticed that we've been using the wrong `gray-10` color all this time (marwa-raging) 🤯 

It's a slight difference, but it's a difference.

### CODE REVIEW

**Comment by marcelgerber** [comment]
Site screenshots are super helpful here!

**Comment by ikesau** [comment]
Added (and integrated) more variables that are being used in Marwa's projects now. I had added them in my catalog branch but this is a better place to do it.

image

File: packages/@ourworldindata/components/src/LabeledSwitch/LabeledSwitch.scss [hunk 1]
```diff
@@ -1,9 +1,6 @@
-$dark-text: #5b5b5b;
```
Comment (ikesau) [inline packages/@ourworldindata/components/src/LabeledSwitch/LabeledSwitch.scss:1]: I can revert these "convention changes" I made by swapping out the top level definitions for inline usage of the global variables if you'd prefer, Marcel.

Comment (marcelgerber) [reply packages/@ourworldindata/components/src/LabeledSwitch/LabeledSwitch.scss:1]: No, I've done more of these now :)

Overall, I find it nice sometimes that colors are given a more meaningful name with local SCSS variables inside a component. I don't mind that, but I do mind using hex codes when color names are available and make more sense.

**Comment by marcelgerber** [comment]
I've continued your work, Ike, and have replaced references to colors with their names in almost all places I could find.

As a sanity check, I also diffed the generated CSS (owid.css) before and after: The only change is using the correct `gray-10` (`#f7f7f7 -> #f2f2f2`), which changed in 27 places. All other CSS lines stayed the same.

**Pull request merged**